packages/rrweb/src/record/observers/canvas/canvas-manager.ts

```diff
@@ -394,7 +396,6 @@ export class CanvasManager implements CanvasManagerInterface {
     canvasElement?: HTMLCanvasElement,
   ) {
     const timeBetweenSnapshots = 1000 / fps;
-    let lastSnapshotTime = 0;
```

Comment:
subtle, i guess when it's called more than once (user is interacting with the page? or no... when our mobile replays are loading we can play/pause really quickly!) it gets cleared out and the new set of closures start at 0 again.

Reply:
:shipit: